cns/restserver/internalapi_test.go

```diff
@@ -537,13 +545,13 @@ func TestReconcileNCWithSystemPods(t *testing.T) {
 	req := generateNetworkContainerRequest(secondaryIPConfigs, uuid.New().String(), "-1")
 
 	expectedAssignedPods := make(map[string]cns.PodInfo)
-	expectedAssignedPods["10.0.0.6"] = cns.NewPodInfo("", "", "customerpod1", "PodNS1")
+	expectedAssignedPods["10.0.0.6"] = cns.NewPodInfo("some-guid-1", "abc-eth0", "customerpod1", "PodNS1")
 
 	// Allocate non-vnet IP for system  pod
-	expectedAssignedPods["192.168.0.1"] = cns.NewPodInfo("", "", "systempod", "kube-system")
+	expectedAssignedPods["192.168.0.1"] = cns.NewPodInfo("some-guid-2", "def-eth0", "systempod", "kube-system")
```

Comment:
@neaggarwMS @rbtr is this a valid scenario? when would we see a host networking pod in the reconcile flow of CNS?

Reply:
> Was this perhaps simulating the reconcile from kubernetes directly? 

yes, the podinfo is constructed using the first two fields or the last two fields depending on if we are rebuilding state from k8s or from CNI.

> Will we still support that scenario?

we have to keep it until we roll the cni-installer init container out to swift, then we will [remove it completely](https://github.com/Azure/azure-container-networking/pull/2087)